### ISSUE TITLE
Landscape layouts

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#56]
Many users prefer to use landscape mode in apps, and Oppia's player view has good potential to better leverage landscape (e.g. having the content & interactions side-by-side rather than top-on-bottom). Although not a specific necessity for the prototype, supporting landscape layouts would go a long way toward demonstrating a well-rounded and polished user experience.

If this is not completed for the prototype, the app should instead be frozen in portrait mode until proper landscape support can be added.

**Comment by BenHenning** [comment]
Since this affects all screens in the app, it's blocked by #51, #32, #53, #54, #55, #48, #46, #44, #42, #41, #40, #27, #28, #29, #30, #31, #32, #33, #34, #35, #26, #25, #24, #23, and #22. There may be future UI work that require landscape layouts work not linked here, and not all of the above layouts will require landscape-specific work. This can also be done during the development of those features, and in multiple pieces in order to optimize the landscape experience for users.

**Comment by anandwana001** [comment]
@rt4914 After the last gsoc, I think we had covered almost everything for landscape view. 
Can we close this issue?

**Comment by rt4914** [comment]
@anandwana001 Actually that project covered some of the major screen but not all. As per [this document](https://docs.google.com/spreadsheets/d/1-xjel5_Su8dTNOCS4p8QycZ9BXBvCLkyF360elvdUxM/edit#gid=0) there are multiple screens which are pending.

**Comment by theayushyadav11** [comment]
Hi @adhiamboperes, I think this issue should be closed now, as we have landscape support for every screen now.
I have tested each screen(which is reproducible) from this [spreadsheet](https://docs.google.com/spreadsheets/d/1-xjel5_Su8dTNOCS4p8QycZ9BXBvCLkyF360elvdUxM/edit#gid=0)  and things seems to work fine.

**Comment by theayushyadav11** [comment]
> Hi @adhiamboperes, I think this issue should be closed now, as we have landscape support for every screen now.
> I have tested each screen(which is reproducible) from this [spreadsheet](https://docs.google.com/spreadsheets/d/1-xjel5_Su8dTNOCS4p8QycZ9BXBvCLkyF360elvdUxM/edit#gid=0)  and things seems to work fine.

Follow up: Issue description has been updated and this issue should only be closed when all 3 tasks are completed.
Thanks

**Comment by adhiamboperes** [comment]
@ShankhanilSaha, please work on this issue as #5972 is already assigned to someone else.

**Comment by ShankhanilSaha** [comment]
> [@ShankhanilSaha](https://github.com/ShankhanilSaha), please work on this issue as [#5972](https://github.com/oppia/oppia-android/issues/5972) is already assigned to someone else.

Sure, will give this issue a thorough look and update you.

**Comment by ShankhanilSaha** [comment]
I read through the issue and its related PR's, comments. Would love to work on this, facing some bazel issues and as soon as I fix it gonna get to it,  assign it to me.

> [@ShankhanilSaha](https://github.com/ShankhanilSaha), please work on this issue as [#5972](https://github.com/oppia/oppia-android/issues/5972) is already assigned to someone else.

**Comment by adhiamboperes** [comment]
@ShankhanilSaha, please feel free to post a github discussion with the Bazel issues that you are facing.

**Comment by ShankhanilSaha** [comment]
> [@ShankhanilSaha](https://github.com/ShankhanilSaha), please feel free to post a github discussion with the Bazel issues that you are facing.

Thanks for assigning the issue, i have opened a discussion regarding my issue.

**Comment by ShankhanilSaha** [comment]
Hi @adhiamboperes , should i raise 3 separate PR's for each task listed in the issue as mentioned in the "How to Contribute" section or just one PR solving them all?

**Comment by theayushyadav11** [comment]
> Hi @adhiamboperes , should i raise 3 separate PR's for each task listed in the issue as mentioned in the "How to Contribute" section or just one PR solving them all? 

Hi @ShankhanilSaha , for this issue you can create a single PR by removing the stuffs and write the requested tests in the same PR itself.
Feel free to reach out if you face any difficulty.

**Comment by ShankhanilSaha** [comment]
Hi there @theayushyadav11 , I have made the necessary changes and writen then tests. However while running the tests i managed to run all other tests other than the ones I wrote in  `app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt` as from the logs seems like Bazel can't recognize that file as a valid target . How should i run those tests?
This is the command i am using to run the tests present in StateFragment.kt
`bazel test //app:src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest` 
I ran the others in this format, they seem to be working fine.

**Comment by theayushyadav11** [comment]
Hi  @ShankhanilSaha , for running the tests you can use `bazel test -- path-to-your-file` command by removing `.kt` in the terminal, for your case it would be: 
```bash
bazel test -- app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest
```
Please refer to [this](https://github.com/oppia/oppia-android/wiki/Bazel-User-Guide#running-tests) documentation on running tests.

**Comment by ShankhanilSaha** [comment]
@theayushyadav11 one of the tests mentioned in task 2 fails, I will be raising a PR of my changes as i have made all the mentioned required changes?
